framework/test/unit/modules/interoperability/base_interoperability_module.spec.ts

```diff
@@ -611,32 +613,12 @@ must NOT have more than ${MAX_NUM_VALIDATORS} items`,
 				},
 			];
 
-			const context = createInitGenesisStateContext(
-				{
-					...genesisInteroperability,
-					chainInfos: [
-						{
-							...chainInfoLocal,
-						},
-						{
-							...chainInfoLocal,
-							chainID: Buffer.from([0, 0, 0, 2]),
-							chainData: {
-								...chainInfoLocal.chainData,
-								name: `${chainData.name}_1`,
-							},
-						},
-					],
-					terminatedStateAccounts,
-				},
-				params,
+			interopModPrototype._verifyTerminatedStateAccountsCommon(
+				terminatedStateAccounts,
+				getMainchainID(params.chainID as Buffer),
 			);
-
-			await interopMod.initGenesisState(context);
-			// `_verifyChainID` is first called from `_verifyChainInfos` from mainchain::module.ts
-			// & size of chainInfos is (has to be the) same as size of terminatedStateAccounts
-			expect(interopMod['_verifyChainID']).toHaveBeenCalledTimes(
-				terminatedStateAccounts.length * 2,
+			expect(interopModPrototype['_verifyChainID']).toHaveBeenCalledTimes(
```

Comment:
I guess it's ok now. @ishantiw please have a look as well since I am not fully comfortable with verifying the correctness of unit tests.